### PR TITLE
Introduce CICD 

### PR DESCRIPTION
**Pull request by jyasveer**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,3 @@
...
-WORKDIR /var/www/html
-
-EXPOSE 8080 8443
-
-USER apache
-
-ENTRYPOINT ["/opt/scripts/run.sh"]
-CMD ["apache"]
+# Builds a Docker to deliver dist/
+FROM nginx:latest
```
Comment (humaton) [inline Dockerfile]: This seems to be Debian based docker image. I had an impression that we are using centos based images for fabric8

File: config/webpack.prod.js [hunk 2]
```diff
@@ -284,38 +166,35 @@ module.exports = webpackMerge(commonConfig, {
            * See: https://github.com/wbuchwalter/tslint-loader
            */
           tslint: {
-            emitErrors: true,
-            failOnHint: true,
+            emitErrors: false,
```
Comment (humaton) [inline config/webpack.prod.js]: Why are you disabling this?

File: install.sh [hunk 3]
```diff
@@ -8,7 +8,12 @@ HTTPD_WELCOME="/etc/httpd/conf.d/welcome.conf"
 INSTALL_PKGS="highlight httpd nss_wrapper gettext";
 
 # Setup necessary packages
-yum -y install epel-release && yum -y install ${INSTALL_PKGS};
+yum -y install epel-release && yum -y install ${INSTALL_PKGS} bzip2;
+yum -y update;
+rpm -ivh https://kojipkgs.fedoraproject.org//packages/http-parser/2.7.1/3.el7/x86_64/http-parser-2.7.1-3.el7.x86_64.rpm && yum -y install nodejs
```
Comment (humaton) [inline install.sh]: Is this really a good idea? Shouldn't we be using SCL here?

Comment (jpopelka) [reply install.sh]: This was a problem in CentOS with http-parser being removed from EPEL, see https://bugzilla.redhat.com/show_bug.cgi?id=1481470
It should be fixed now, see also https://github.com/fabric8-analytics/fabric8-analytics-worker-base/pull/10

Comment (humaton) [reply install.sh]: @jyasveer so this line can be removed.

**Comment by jyasveer** [comment]
@humaton Could you see this now ?

**Comment by humaton** [comment]
LGTM

**Pull request merged**